### PR TITLE
Added support for -excludeLibs command line option

### PR DESCRIPTION
**Pull request by saidinesh5**
This helps us avoid pesky libraries which are optional by the main
binary but may be dragged in by some other libraries like
https://github.com/probonopd/linuxdeployqt/issues/235

Usage: -exclude-libs=libqsqlibase,libqsqlodbc,libqsqlpsql,libqsqltds

### CODE REVIEW

File: tools/linuxdeployqt/main.cpp [hunk 1]
```diff
@@ -82,6 +82,8 @@ int main(int argc, char **argv)
         qInfo() << "   -no-copy-copyright-files : Skip deployment of copyright files.";
         qInfo() << "   -extra-plugins=<list>    : List of extra plugins which should be deployed,";
         qInfo() << "                              separated by comma.";
+        qInfo() << "   -exclude-libs=<list>     : List of extra plugins which should be deployed,";
```
Comment (TheAssassin) [inline tools/linuxdeployqt/main.cpp]: Apparently a copy-paste error.

**Comment by TheAssassin** [comment]
@saidinesh5 would you like to update your PR? We could merge it then.

**Comment by probonopd** [comment]
Adding this option will almost certainly result in broken AppImages...

**Comment by TheAssassin** [comment]
@probonopd I don't think so, people who skip libraries most likely know what they do. We could print a warning though, like, "Excluding the following libraries. WARNING: this might break the AppImage, so please double-check the list."

**Comment by probonopd** [comment]
Excellent idea @saidinesh5

**Comment by TheAssassin** [comment]
Thanks!

**Pull request merged**